CoreScriptsRoot/Modules/DeveloperConsoleModule.lua

```diff
@@ -2219,6 +2229,22 @@ end
 ----------
 -- Tabs --
 ----------
+function Methods.GetCurrentOpenTab(devConsole)
+	local tabs = devConsole.Tabs
+	if tabs == nil then 
+		return nil
+	end
+
+	for i = 1, #tabs do
+		local tab = tabs[i]
+		if tab.Open then
```

Comment:
it is interesting that it could be possible to have more than one open tab... at least in theory. Would it make more sense for open tab to be a singleton value instead of a parameter on tab?